openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1065,70 +1096,91 @@ Authorization: BEARER czZCaGRSa3F0MzpnWDFmQmF0M2JW
              "UniversityDegreeCredential"
            ]
          },
-         "proof":{
-            "proof_type":"jwt",
-            "jwt":"eyJ0eXAiOiJvcGVuaWQ0dmNpL...Lb9zioZoipdP-jvh1WlA"
-         }
+         "proofs": [
+            {
+               "proof_type":"jwt",
+               "jwt":"eyJ0eXAiOiJvcGVuaWQ0dmNpL...Lb9zioZoipdP-jvh1WlA"
+            },
+            {
+               "proof_type":"jwt",
+               "jwt":"eyJraWQiOiJkaWQ6ZXhhbXBsZ...KPxgihac0aW9EkL1nOzM"
+            }
+         ]
       },
       {
-         "format":"mso_mdoc",
-         "doctype":"org.iso.18013.5.1.mDL",
-         "proof":{
-            "proof_type":"jwt",
-            "jwt":"eyJraWQiOiJkaWQ6ZXhhbXBsZ...KPxgihac0aW9EkL1nOzM"
-         }
+         "credential_identifier":"CivilEngineeringDegree-2023",
+         "proofs": [
+            {
+               "proof_type":"jwt",
+               "jwt":"eyJraWQiOiJkaWQ6ZXhhbXBsZ...KPxgihac0aW9EkL1nOzM"
+            }
+         ]
+      },
+      {
+         "credential_identifier":"ElectricalEngineeringDegree-2023",
```

Comment:
this seems like a type, is this supposed to match up with `vct` ?

Reply:
In the new terminology it is called an identifier for a Credential Dataset, so it references a dataset for a particular Credential Configuration. For example, if you requested a diploma from university, they could offer you all diplomas that you requested, all of the same "type" but with different claim values. The identifier  could then be (optionally) used in the Credential Request